build.gradle

```diff
@@ -46,6 +46,10 @@ dependencies {
   // ─────────────────────────────────────────────────────────────────────────
   // https://repo.enonic.com/service/rest/repository/browse/public/com/enonic/lib/
 
+  // https://market.enonic.com/vendors/enonic/lib-asset
```

Comment:
The entire PR is just adding some comments, I'd it correct?

Reply:
Yes